rfcs/fetch_json_et_al.md

```diff
@@ -0,0 +1,122 @@
+# RFC 177: `fetch_json` and other `ShadowRealm` facilities
+
+## Summary
+
+This follows up [RFC 107][rfc-107].
+[`ShadowRealm`s][proposal] are a new sandboxing primitive, currently in
+the process of being integrated into relevant Web specifications.
+`ShadowRealm` environments resemble vanilla JS shell environments, but
+do have some Web interfaces available, such as `URL`.
+
+I've submitted a [pull request][pr] with some changes to the
+test harness.
+The goal is to more easily allow existing WPT test files to run in
+`ShadowRealm` contexts, without writing separate duplicated tests for
+`ShadowRealm`.
+
+The most notable change is a `fetch_json` function to allow importing
+test data from JSON files across the sandboxing boundary.
+
+## Details
+
+The [pull request][pr] comprises four commits, which I'll explain in
+detail in this section.
+
+### Document self.GLOBAL.isShadowRealm()
```

Comment:
If you want to land this fix right away and drop it from this RFC that would be fine.

Reply:
Thanks, I may split it out if I get a chance before the RFC finishes.